src/commands/codepush/lib/react-native-utils.ts

```diff
@@ -485,10 +485,21 @@ async function getHermesCommandFromGradle(gradleFile: string): Promise<string> {
   }
 }
 
-export function getAndroidHermesEnabled(gradleFile: string): boolean {
-  return parseBuildGradleFile(gradleFile).then((buildGradle: any) => {
-    return Array.from(buildGradle["project.ext.react"] || []).some((line: string) => /^enableHermes\s{0,}:\s{0,}true/.test(line));
-  });
+export async function getAndroidHermesEnabled(gradleFile: string): Promise<boolean> {
+  // find hermesEnabled in gradle.properties first
+  try {
+    const properties = await g2js.parseFile(path.join("android", "gradle.properties"));
```

Comment:
what happens if the file doesn't not exist? Will it crash? If yes, then it won't try to parse `gradleFile`, which is not what we would want.

Reply:
Sure.